src/gordon_gcp/plugins/service/event_consumer.py

```diff
@@ -324,7 +335,7 @@ def __init__(self, config, success_channel, error_channel, metrics,
 
     def _create_gevent_msg(self, pubsub_msg, data, schema):
         log_entry = f'Created a "{schema}" message.'
-        msg = GEventMessage(pubsub_msg, data)
+        msg = GEventMessage(pubsub_msg, data, phase=self.start_phase)
```

Comment:
yup 👍

Reply:
But actually now that I think about maybe this message handling should be in the GEventMessage emit too?